src/components/FormRenderer/components/FormTemplate/index.tsx

```diff
@@ -54,11 +58,11 @@ const FormTemplate: FunctionComponent<FormTemplateProps> = ({
     );
 
     const actionsVisible = useMemo(() => {
-        return !(schema.fields && schema.fields.length > 0 && schema.fields[0].component === componentTypes.WIZARD);
+        return !(schema.fields.length > 0 && schema.fields[0].component === componentTypes.WIZARD);
     }, [schema.fields]);
 
     return (
-        <Form header={schema.header} description={schema.description} actions={actionsVisible ? actions : undefined}>
+        <Form header={schema.title} description={schema.description} actions={actionsVisible ? actions : undefined}>
```

Comment:
Please keep  the *header*

Reply:
@jessieweiyi this is also come from `Schema` type:
```js
interface Schema {
  title?: ReactNode;
  description?: ReactNode;
  fields: Field[];
}
```